mllib/src/main/scala/org/apache/spark/ml/tree/impl/GradientBoostedTrees.scala

```diff
@@ -261,7 +264,8 @@ private[spark] object GradientBoostedTrees extends Logging {
       boostingStrategy: OldBoostingStrategy,
       validate: Boolean,
       seed: Long,
-      featureSubsetStrategy: String): (Array[DecisionTreeRegressionModel], Array[Double]) = {
+      featureSubsetStrategy: String,
+      instr: Option[Instrumentation]): (Array[DecisionTreeRegressionModel], Array[Double]) = {
```

Comment:
nit: I suggest to add a default value   `instr: Option[Instrumentation] = None`,
so that changes in mllib/tree/GradientBoostedTrees.scala and ml/tree/impl/GradientBoostedTreesSuite.scala can be avoided.

Reply:
Good point. I will update the code.